starboard/shared/ffmpeg/BUILD.gn

```diff
@@ -18,18 +18,29 @@ ffmpeg_specialization_sources = [
   "ffmpeg_common.h",
   "ffmpeg_demuxer_impl.cc",
   "ffmpeg_demuxer_impl.h",
-  "ffmpeg_video_decoder_impl.cc",
-  "ffmpeg_video_decoder_impl.h",
 ]
 
+if (!defined(is_raspi)) {
+  ffmpeg_specialization_sources += [
+    # TODO(b/291783511): This code is unimplemented on certain platforms.
```

Comment:
Should this be b/208910380?

Reply:
It's kinda both—if your bug is fixed, this compiles, if the bug I have in the todo is fixed, it doesn't need to compile. We really want to fix both either way, but I have no hard pref on which is in the TODO

Reply:
I opened b/292127297 yesterday which is also deals with the same stuff. @sideb0ard is investigating.

It sounded like this should all go away but I don't know what the timeline is.